### ISSUE TITLE
Napari viewer list 0 coordinates

### ISSUE THREAD

**Issue by pconesa** [scipion-em/scipion-em-tomo#420]
Reported by a user

**Comment by azazellochg** [comment]
Ehm, a bit more information would be useful...

**Comment by pconesa** [comment]
@plosana ?

**Comment by pconesa** [comment]
I understood it is the this of tomograms ?

**Comment by plosana** [comment]
Yes sorry I had not seen this issue in github. I think it's an internal problem of my machine, I had some internal library problems and after git pulling some plugins broke and I have had to reinstall them. After reinstall it seems to work fine. In case it happens to someone else, my problem was located on the Set of Coordinates visualisation, when I wanted to see coordinates linked to a set of tomograms, instead of the real number of coordinates I saw 0 for every tomogram:
![image](https://github.com/scipion-em/scipion-em-tomo/assets/8470316/6ce04611-a1bf-4b31-b9b0-d9e37bc83994)
but after reinstalling if detects them fine. I think that @Vilax had a similar problem. Thank you everyone :)
